File: JavaScript/Simple chotu calculator.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

function calculator() {
    rl.question("Enter first number: ", (aInput) => {
        const a = parseFloat(aInput);
        
        rl.question("Enter second number: ", (bInput) => {
            const b = parseFloat(bInput);
            
            rl.question("Enter the operation (+, -, *, /, %): ", (op) => {
                let result;
                
                switch(op) {
                    case '+':
                        result = a + b;
                        console.log(`${a} + ${b} = ${result}`);
                        break;
                        
                    case '-':
                        result = a - b;
                        console.log(`${a} - ${b} = ${result}`);
                        break;
                        
                    case '*':
                        result = a * b;
                        console.log(`${a} * ${b} = ${result}`);
                        break;
                        
                    case '/':
                        if (b === 0) {
                            console.log("Error: Division by zero is not allowed!");
                        } else {
                            result = a / b;
                            console.log(`${a} / ${b} = ${result}`);
                        }
                        break;
                        
                    case '%':
                        if (b === 0) {
                            console.log("Error: Modulus by zero is not allowed!");
                        } else {
                            result = Math.floor(a) % Math.floor(b);
                            console.log(`${Math.floor(a)} % ${Math.floor(b)} = ${result}`);
                        }
                        break;
                        
                    default:
                        console.log("Invalid Input! Please use +, -, *, /, or %");
                }
                
                rl.close();
            });
        });
    });
}

calculator();

File: Python/Simple chotu calculator.py
def calculator():
    try:
        a = float(input("Enter first number: "))
        b = float(input("Enter second number: "))
        op = input("Enter the operation (+, -, *, /, %): ")

        if op == '+':
            print(f"{a} + {b} = {a + b}")
        elif op == '-':
            print(f"{a} - {b} = {a - b}")
        elif op == '*':
            print(f"{a} * {b} = {a * b}")
        elif op == '/':
            if b == 0:
                print("Error: Division by zero is not allowed!")
            else:
                print(f"{a} / {b} = {a / b}")
        elif op == '%':
            if b == 0:
                print("Error: Modulus by zero is not allowed!")
            else:
                print(f"{int(a)} % {int(b)} = {int(a) % int(b)}")
        else:
            print("Invalid Input! Please use +, -, *, /, or %")
    
    except ValueError:
        print("Please enter valid numbers!")

if __name__ == "__main__":
    calculator()